src/main/resources/layouts/report.fxml

```diff
@@ -77,6 +78,11 @@
                               </Label>
                            </children>
                         </HBox>
+                        <HBox prefHeight="26.0" prefWidth="210.0">
+                           <children>
+                              <Button fx:id="expandCollapseButton" mnemonicParsing="false" prefHeight="25.0" prefWidth="79.0" text="Collapse" />
```

Comment:
Can you have a look on a better position of the button?
![image](https://user-images.githubusercontent.com/5772511/205905148-828f9c07-f889-4471-bc7d-075b440797dd.png)
currently it is not clear what the button will "expand"

Reply:
Changed the text to "Expand Report list"

Reply:
That is already better. But the position is not good / the context "report list" should not be needed to be stated.
Please have a look at the basic design principals:
- https://www.youtube.com/watch?v=YkXRt9kMfzU
- https://achtung-designer.com/gestaltgesetze-die-designs-besser-machen/
They are very helpful to know (at least for the end users if they were applied) :)

Reply:
reworked it a bit 

![grafik](https://user-images.githubusercontent.com/86607279/207633350-7de6dd66-95df-4260-b182-9b562435eccb.png)